### ISSUE TITLE
Magic numbers

### ISSUE THREAD

**Issue by svgeesus** [w3c/PNG-spec#33]
The [PNG signature](https://w3c.github.io/PNG-spec/#5PNG-file-signature) is listed in decimal, which is unusual:

`137 80 78 71 13 10 26 10`

We should probably add the more usual hex representation:

`89 50 4E 47 0D 0A 1A 0A`

**Comment by ProgramMax** [comment]
Agreed. Without careful inspection, I assumed hex while scanning with my eyes.
Perhaps in addition to using hex numbers, the text can explicitly say the values are hexadecimal.

**Comment by randy408** [comment]
Every chunk's type field is also listed in decimal.

**Comment by svgeesus** [comment]
> Every chunk's type field is also listed in decimal.

They are, which is odd, but doesn't affect anything but PNG.

General file signatures are typically in hex, so it makes sense to add the hex values as well. I initially made the same mistake as @ProgramMax when I was filling in the updated IANA registration for `image/png` which needs any magic numbers. I was alerted because 

1. The value 137
2. Remembering that the signature includes `cr` `lf`
